packages/bits/src/lib/select-v2/option-key-control.service.ts

```diff
@@ -1,9 +1,10 @@
-import { ActiveDescendantKeyManager } from "@angular/cdk/a11y";
+import { ActiveDescendantKeyManager, LiveAnnouncer } from "@angular/cdk/a11y";
```

Comment:
It might be good to add some unit tests for the changes in this file.

Reply:
we will get rid of keyCode property across all project (NUI-2492). I guess ut for this file better create in scope of this ticket avoid additional work.